test/functional/rpc_fundrawtransaction.py

```diff
@@ -485,7 +485,9 @@ def run_test(self):
         rawtx = self.nodes[2].createrawtransaction(inputs, outputs)
         fundedTx = self.nodes[2].fundrawtransaction(rawtx)
 
-        signedTx = self.nodes[2].signrawtransactionwithwallet(fundedTx['hex'])
+        blindedTx = self.nodes[2].blindrawtransaction(fundedTx['hex'])
+        signedTx = self.nodes[2].signrawtransactionwithwallet(blindedTx)
+        #fail next line: test_framework.authproxy.JSONRPCException: Transaction output has nonce, but is not blinded. Did you forget to call blindpsbt, blindrawtranssaction, or rawblindrawtransaction? (-25)
```

Comment:
Oh, no, that comment should go away.

This was already broken before my PR, because addresses are confidential by default, but the test was not calling blindrawtransaction. It was caught by my PR because I added the "no bullshit nonces left" check in sendrawtransaction, which caught the transaction in the test going out unblinded.

Reply:
Fixed.